atomic_reactor/plugins/pre_pin_operator_digest.py

```diff
@@ -302,6 +302,11 @@ def run_in_worker(self):
             self.log.info("Creating relatedImages section in %s", operator_csv.path)
             operator_csv.set_related_images()
 
+            operator_csv_modifications = self._fetch_operator_csv_modifications()
+            if operator_csv_modifications:
+                operator_csv.modifications_append(operator_csv_modifications.get('append'))
```

Comment:
if you'd use `get(x, {})` you don't have to add `if not ...` to every `modifications_*` method, because it's would be an empty iteration.

Reply:
ACK